src/pykeen/pipeline/api.py

```diff
@@ -1366,6 +1366,8 @@ def _safe_evaluate(
     :raises RuntimeError:
         If CUDA ran into OOM issues trying to evaluate the model on the hardware at hand with the given parameters.
     """
+    if not mapped_triples.shape[0]:
+        return MetricResults(data={})
```

Comment:
I am not sure why I can instantiate the class, even though it has an abstract method 🤔 

anyway, this is a hacky solution, I do not want to keep 😅